strawberryfields/backends/bosonicbackend/backend.py

```diff
@@ -401,20 +551,20 @@ def measure_heterodyne(self, mode, shots=1, select=None):
         return np.array([[res]])
 
     def prepare_gaussian_state(self, r, V, modes):
-        if isinstance(modes, int):
+        if np.isinstance(modes, int):
             modes = [modes]
 
-        # make sure number of modes matches shape of r and V
+        # make sure number of modes matches np.shape of r and V
         N = len(modes)
         if len(r) != 2 * N:
             raise ValueError("Length of means vector must be twice the number of modes.")
-        if V.shape != (2 * N, 2 * N):
+        if V.np.shape != (2 * N, 2 * N):
```

Comment:
I am not sure one can do `array.np.shape` ?

Reply:
```suggestion
        if V.shape != (2 * N, 2 * N):
```